### DISCUSSION TITLE
GPS data anonymised

### DISCUSSION THREAD

**Discussion by fnzv** [bimmerconnected/bimmer_connected#575] [Q&A] [answered]
Hi!
Would it be possible to add a specific flag into the library in order to de-anonymoise the GPS data?
Or it's already anonymised by the API itself so it cannot be done

I am working on [this exporter ](https://github.com/fnzv/bimmer_exporter )and i wanted to create a real time map on Grafana for this data

Thank you

**Comment by rikroe** [comment, marked as answer]
Cool to see how other people use the library :)

Use `status` with the `-j` argument instead of `fingerprint` and you'll get the non-redacted data on stdout: https://bimmer-connected.readthedocs.io/en/latest/cli.html#named-arguments

One note though: the CLI is not built for recurring data retrieval (yet). It everytime goes through a full login cycle instead of using the stored OAuth refresh tokens & vehicle, so you might run into some rate limiting issues if querying too often.

This would not happen if you're able to call the library directly from Python and store the current `MyBMWAccount` object inside your server. Don't know it this is possible from Go, tough.

**Comment by fnzv** [comment]
Thank you, i should have read the docs a little bit more!
I will write it in my notes
